storage/innobase/xtrabackup/test/t/remote_tablespaces.sh

```diff
@@ -33,14 +45,18 @@ do
     $MYSQL $MYSQL_ARGS test <<EOF
 CREATE TABLE p$i(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
   DATA DIRECTORY = '$undo_dir' ENGINE=InnoDB;
+CREATE TABLE r$i(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
+  TABLESPACE TS1 ENGINE=InnoDB;
+CREATE TABLE s$i(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
+  TABLESPACE TS2 ENGINE=InnoDB;
 EOF
 done
 
 # create some undo tablespaces
-mysql -e "CREATE UNDO TABLESPACE rundo1 ADD DATAFILE '$remote_dir/undo1.ibu'"
-mysql -e "CREATE UNDO TABLESPACE rundo2 ADD DATAFILE '$remote_dir/undo2.ibu'"
-mysql -e "CREATE UNDO TABLESPACE uundo1 ADD DATAFILE '$undo_dir/undo1.ibu'"
-mysql -e "CREATE UNDO TABLESPACE uundo2 ADD DATAFILE '$undo_dir/undo2.ibu'"
+mysql -e "CREATE UNDO TABLESPACE rundo1 ADD DATAFILE '$remote_dir/rundo1.ibu'"
+mysql -e "CREATE UNDO TABLESPACE rundo2 ADD DATAFILE '$remote_dir/rundo2.ibu'"
+mysql -e "CREATE UNDO TABLESPACE uundo1 ADD DATAFILE '$undo_dir/uundo1.ibu'"
+mysql -e "CREATE UNDO TABLESPACE uundo2 ADD DATAFILE '$undo_dir/uundo2.ibu'"
```

Comment:
why undo tablespaces creation is changed?

Reply:
This test was failing because of PXB-2099. I clubbed it with  this changes.

Reply:
Ok, how did you miss this? So from now onwards, Please provide jenkins link and failure analysis for every PR

Reply:
agree 100%. it makes even my life easy.